CHANGELOG.rst

```diff
@@ -59,7 +59,7 @@ Nieuwe features
   observability.
 * [:taiga-us:`3408`, :pr:`1881`]: Nieuw selectiescherm voor eHerkenning-inlog met zoek-
   en dropdownfunctie voor betere UX waarmee gewiseld kan worden tussen vestigingen.
-* [:taiga-us:`3450`, :taiga-ta:`3454`]: CKEditor wordt vervangen door Prosemirror
+* [:taiga-us:`3450`, :taiga-ta:`3454`, :pr:`2067`]: CKEditor wordt vervangen door Prosemirror
```

Comment:
Eigenlijk zou dit "IS vervangen door..." moeten zijn, maar het is een oude tekst, laat maar staan, maar "ackshually" zou het alleen moeten gaan over wat er is opgeleverd, dus zou in 'voltooid verleden tijd' of 'tegenwoordige tijd' moeten.